ui/webui/src/components/installation/InstallationProgress.jsx

```diff
@@ -97,31 +123,40 @@ export class InstallationProgress extends React.Component {
         }
 
         return (
-            <Bullseye className={idPrefix + "-status-" + status}>
-                <EmptyStatePanel
-                  icon={icon}
-                  loading={!icon}
-                  paragraph={
-                      status !== "success"
-                          ? <Progress
-                              id={idPrefix + "-bar"}
-                              label={cockpit.format("$0 of $1", step, steps)}
-                              max={steps}
-                              min={0}
-                              title={message}
-                              value={step}
-                              valueText={cockpit.format("$0 of $1", step, steps)}
-                              variant={status}
-                          />
-                          : <Text>{_("Fedora is successfully installed. To begin using it, reboot your system.")}</Text>
-                  }
-                  secondary={
-                      status === "success" &&
-                      <Button onClick={exitGui}>{_("Reboot")}</Button>
-                  }
-                  title={title}
-                />
-            </Bullseye>
+            <EmptyStatePanel
+              icon={icon}
+              loading={!icon}
+              paragraph={
+                  <ProgressStepper isCenterAligned>
+                      {progressSteps.map((progressStep, index) => {
+                          let variant = "pending";
+                          const ariaLabel = "pending step";
```

Comment:
aria labels should get translated normally. It's read by screen readers.

Reply:
Thanks, I was wondering what all they are used for. BTW, I think we will need to add them in more places later on.

Reply:
Anyway, fixed. :)